packages/extension-seo-url-mapping/src/types/config.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import type {ApplicationExtensionConfig} from '@salesforce/pwa-kit-extension-sdk/types'
+
+// Defines the map of resource types to component display name.
+// https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-seo?meta=getUrlMapping
+type ResourceTypeToComponentMap = {
+    category: string
+    product: string
+    content_asset: string
+}
```

Comment:
We might want to leverage the enum that the API uses or the response type and gave those values.. we know that the key might be an entry of the enum = CATEGORY | PRODUCT | CONTENT_ASSET which would hep clean up this type.

Reply:
We also still have the problem where we are mapping to the components display name. So we'll want to harden that solution a little. Maybe the pages in the storefront and store locator need to be prefixed with the extensions name/id to help prevent collisions or duplicate component/page names which might cause this solution to fall flat.